go/vt/vtgate/engine/fake_primitive_test.go

```diff
@@ -118,9 +119,9 @@ func (f *fakePrimitive) TryStreamExecute(vcursor VCursor, bindVars map[string]*q
 
 	return nil
 }
-func (f *fakePrimitive) GetFields(vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
+func (f *fakePrimitive) GetFields(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
 	f.log = append(f.log, fmt.Sprintf("GetFields %v", printBindVars(bindVars)))
-	return f.TryExecute(vcursor, bindVars, true /* wantfields */)
+	return f.TryExecute(context.Background(), vcursor, bindVars, true)
```

Comment:
This is testing code, but shouldn't we use `ctx` here instead of `context.Background()`